packages/editor/src/components/post-author/index.js

```diff
@@ -63,4 +64,5 @@ export default compose( [
 		},
 	} ) ),
 	withInstanceId,
+	withFilters( 'editor.PostAuthor' ),
```

Comment:
Like we discussed in previous #core-js chats, we should resist adding new filters/actions.
An alternative here would be to build a custom edit-post module.

Reply:
It is part of the `Post & Status` panel, so it had to be re-created from scratch. I think we should introduce all missing filters for the elements we put in the sidebar for backward compatibility and consistency. However, we should stop doing it in other places unless there is a really good reason to do so. I can't think of any at the moment :)